scripts/warroom/emergency_swap.py

```diff
@@ -0,0 +1,35 @@
+from rich.console import Console
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+console = Console()
+
+prod = False
+
+
+def main(symbol_in, symbol_out, amount=None, use_3pool="true"):
+    vault = GreatApeSafe(r.badger_wallets.treasury_vault_multisig)
+
+    tokens = {
+        "usdc": vault.contract(r.treasury_tokens.USDC),
+        "usdt": vault.contract(r.treasury_tokens.USDT),
+        "dai": vault.contract(r.treasury_tokens.DAI),
+    }
+
+    token_in = tokens[symbol_in]
+    token_out = tokens[symbol_out]
+    amount = token_in.balanceOf(vault) if not amount else int(amount)
```

Comment:
i would suggest to add the decimals of the respective token here programmatically, so that the user can pass `100_000` as an amount an not have to worry about whether it is 6 or 18 decimals.

Reply:
good idea